Add chmod and log-file options

esalberg opened the pull request.

esalberg commented:
I reverted previous changes to .fixtures.yml (change git:// to https://) since that doesn't seem to be needed anymore?  If it is, I can go back and put the change back in.

eputnam commented:
@esalberg , thanks for this! please rebase and add a test or two and we'd be glad to merge!

esalberg commented:
Rebased and added tests for chown, chmod, and logfile by copying the other get tests.

Travis is showing rake errors based on no method last_comment, which from a quick search could be due to the rake / rspec version.  Is this something I need to fix?

eputnam commented:
@esalberg thanks! i've asked for a revision to #88 and then we'll be set to merge. If you'd like to put up another PR to unpin rspec yourself, you're welcome, but we'd like to see it done separately from this work.

esalberg commented on manifests/get.pp:
```diff
@@ -115,6 +120,14 @@
   }
 
+  if $chmod {
+    $myChmod = "--chmod=${chmod}"
```
Whoops, sorry about that.  Done.

I think my Puppet Lint updated since I last committed changes to get.pp - it's complaining about variables containing uppercase letters.  I updated the ones I added not to contain them, but there are 35 other errors.  Is it okay to go through and update that as part of this PR, or would you prefer for me to disable lint temporarily?

eputnam replied:
go ahead and just update yours for this one.

esalberg commented:
Appreciate the fix on the tests.  Thanks for the merge!
